docs/pages/getting-started/providers/sailpoint.mdx

```diff
@@ -0,0 +1,176 @@
+      name: "Identity Secure Cloud",
+      type: "oauth",
+      clientId: process.env.ISC_CLIENT_ID!,
+      clientSecret: process.env.ISC_CLIENT_SECRET!,
+      authorization: {
+        url: `${process.env.ISC_BASE_URL!}/oauth/authorize`,
+        params: { scope: 'sp:scopes:all' },
+      },
+      token: `${process.env.ISC_BASE_API_URL!}/oauth/token`,
+      userinfo: `${process.env.ISC_BASE_API_URL!}/oauth/userinfo`,
+      profile(profile: IdentitySecureCloudProfile) {
+        return {
+            tenant: profile.tenant,
+            id: profile.id,
+            uid: profile.uid,
+            email: profile.email,
+            phone: profile.phone,
+            workPhone: profile.workPhone,
+            firstname: profile.firstname,
+            lastname: profile.lastname,
+            capabilities: profile.capabilities,
+            displayName: profile.displayName,
+            name: profile.uid,
+        }
+      },
+      style: { text: "#011E69", bg: "#fff", logo: "sailpoint.svg" },
+    },
+  ],
+})
+```
+
+  </Code.Next>
+  <Code.Svelte>
+
+```ts filename="/src/auth.ts"
+import { SvelteKitAuth } from "@auth/sveltekit"
+import { env } from "$env/dynamic/prviate"
+
+export const { handle, signIn, signOut } = SvelteKitAuth({
+  providers: [
+    {
+      id: "identitySecureCloud",
+      name: "Identity Secure Cloud",
+      type: "oauth",
+      clientId: env.ISC_CLIENT_ID!,
+      clientSecret: env.ISC_CLIENT_SECRET!,
+      authorization: {
+        url: `${env.ISC_BASE_URL!}/oauth/authorize`,
+        params: { scope: 'sp:scopes:all' },
+      },
+      token: `${env.ISC_BASE_API_URL!}/oauth/token`,
+      userinfo: `${env.ISC_BASE_API_URL!}/oauth/userinfo`,
+      profile(profile: IdentitySecureCloudProfile) {
+        return {
+            tenant: profile.tenant,
+            id: profile.id,
+            uid: profile.uid,
+            email: profile.email,
+            phone: profile.phone,
+            workPhone: profile.workPhone,
+            firstname: profile.firstname,
+            lastname: profile.lastname,
+            capabilities: profile.capabilities,
+            displayName: profile.displayName,
+            name: profile.uid,
+        }
+      },
+      style: { text: "#011E69", bg: "#fff", logo: "sailpoint.svg" },
+    },
+  ],
+})
+```
+
+  </Code.Svelte>
+  <Code.Express>
+
+```ts filename="/src/app.ts"
+import { ExpressAuth } from "@auth/express"
+
+app.use("/auth/*", ExpressAuth({ providers: [
+  {
+      id: "identitySecureCloud",
+      name: "Identity Secure Cloud",
+      type: "oauth",
+      clientId: process.env.ISC_CLIENT_ID!,
+      clientSecret: process.env.ISC_CLIENT_SECRET!,
+      authorization: {
+        url: `${process.env.ISC_BASE_URL!}/oauth/authorize`,
+        params: { scope: 'sp:scopes:all' },
+      },
+      token: `${process.env.ISC_BASE_API_URL!}/oauth/token`,
+      userinfo: `${process.env.ISC_BASE_API_URL!}/oauth/userinfo`,
+      profile(profile: IdentitySecureCloudProfile) {
```

Comment:
This should be somethign like this, for all 3 examples

```ts
      profile(profile: IdentitySecureCloudProfile) {
        return {
            id: profile.id,
            email: profile.email,
            name: `${profile.firstname} ${profile.lastname}`,
            image: null
        }
      },
```

Reply:
Thanks a lot for the suggestion, already made the changes, I still keep name: profile.uid. The uid value is like yu-chih.chung, which is more useful as the login account name for SailPoint Identity Secure Cloud. I think display name is less usefuly in this case.